coordinator/src/db/routing_fees.rs

```diff
@@ -0,0 +1,69 @@
+use crate::routing_fee;
+use crate::schema::routing_fees;
+use anyhow::Result;
+use bitcoin::hashes::hex::ToHex;
+use diesel;
+use diesel::prelude::*;
+use dlc_manager::ChannelId;
+use hex::FromHex;
+use time::OffsetDateTime;
+
+#[derive(Insertable, Debug, PartialEq)]
+#[diesel(table_name = routing_fees)]
+struct NewRoutingFee {
```

Comment:
🤔 is it really necessary to have a separate insertable model for this simple data?

Personally I would prefer one model as I guess it will also reduce complexity.